StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1120CodeFixProvider.cs

```diff
@@ -53,27 +53,23 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var nodesToRemove = new List<SyntaxTrivia>();
```

Comment:
`var node` above and `nodesToRemove` could be renamed to show that they are about trivia and not actual nodes.